.travis.yml

```diff
@@ -0,0 +1,14 @@
+---
+language: node_js
+node_js:
+  - "6"
```

Comment:
why run travis on node v6? could we skip this and let it fallback to using node version from `.nvmrc`?
https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Specifying-Node.js-versions-using-.nvmrc

Reply:
sure, we can, but we've just set our nvmrc for local development. we can also take advantage of supporting node LTS for the module itself, regardless of what version is used for development.

Reply:
yes, sounds good on having this here. thanks for clarifying :)